syn/syn_setup.example.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+if [ $# -eq 1 ]; then
+  export LR_SYNTH_OUT_DIR=$1
+elif [ $# -eq 0 ]; then
+  export LR_SYNTH_OUT_DIR_PREFIX=syn_out
+  export LR_SYNTH_OUT_DIR=$(date +"${LR_SYNTH_OUT_DIR_PREFIX}_%d_%m_%Y_%H_%M_%S")
+else
+  echo "Usage $0 [synth_out_dir]"
+  exit 1
+fi
+
+export LR_SYNTH_TIMING_RUN=1
+export LR_SYNTH_FLATTEN=1
+
+# SETUP CELL LIBRARY PATH
+# Uncomment the lines below and set the path to an appropriate .lib file
+# export LR_SYNTH_CELL_LIBRARY_PATH=/path/to/NangateOpenCellLibrary_typical.lib
+# export LR_SYNTH_CELL_LIBRARY_NAME=nangate
```

Comment:
Instead of asking people to copy and modify this file you can provide an extension point: use this script to set the variables to some defaults, and then source another script (if it exists) with user-defined values.

Reply:
The issue is there's no defaults that can work. A standard cell library with timing information is required (so the cmos_cells.lib from yosys won't work). We could include the nangate library in our repository but I wasn't certain about the licensing issues around that so thought it was better to ask people to acquire it themselves. We can discuss offline.